tracer/src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -62,12 +62,14 @@ internal class AgentWriter : IAgentWriter
 
         private long _droppedTraces;
 
-        public AgentWriter(IApi api, IStatsAggregator statsAggregator, IDogStatsd statsd, bool automaticFlush = true, int maxBufferSize = 1024 * 1024 * 10, int batchInterval = 100)
-        : this(api, statsAggregator, statsd, MovingAverageKeepRateCalculator.CreateDefaultKeepRateCalculator(), automaticFlush, maxBufferSize, batchInterval)
+        private bool _experimentalAppsecStandaloneEnabled;
```

Comment:
nit: this should be `readonly`. I also think we shouldn't use "experimental" in the name here, this isn't public stuff, and it's how we _intend_ to implement it, so go with it IMO :D